core/modules/query/ValueExpr.cc

```diff
@@ -88,6 +88,29 @@ operator<<(std::ostream& os, ValueExpr::FactorOp const& fo) {
     return os;
 }
 
+void ValueExpr::FactorOp::dbgPrint(std::ostream& os) const {
+    os << "FactorOp(op:";
+    switch(op) {
+    case ValueExpr::NONE: os << "NONE"; break;
+    case ValueExpr::UNKNOWN: os << "UNKNOWN"; break;
+    case ValueExpr::PLUS: os << "PLUS"; break;
+    case ValueExpr::MINUS: os << "MINUS"; break;
+    case ValueExpr::MULTIPLY: os << "MULTIPLY"; break;
+    case ValueExpr::DIVIDE: os << "DIVIDE"; break;
+    default: throw std::runtime_error("unhandled op"); break;
```

Comment:
In other places you just print "!!unhandled!!", is this one much worse and deserves exception?

Reply:
no; I think I did this one first and then decided that throwing was a little extreme and decided to go with "!!unhandled!!", but forgot to change this one. willdo